integration/serviceWorker/serviceWorker.js

```diff
@@ -14,7 +14,7 @@
 * limitations under the License.
 */
 
-importScripts('../../../dist/package/firebase-app.js');
+importScripts('./node_modules/firebase/firebase-app.js');
```

Comment:
why use the node_modules instance of firebase here? Would have thought you'd need this to be dist